app/scripts/filters/resources.js

```diff
@@ -120,16 +120,29 @@ angular.module('openshiftConsole')
       });
       return nameCount;
     }
-    return function (resource, projects){
+    function escapeLessThanGreaterThan(str) {
+      return str.
+        replace(/</g, '&lt;').
+        replace(/>/g, '&gt;');
+    }
```

Comment:
Are we sure there isn't a problem with other content? Need to be careful when setting something as HTML to avoid XSS attacks (although Angular helps sanitize unsafe markup I believe). I'd rather be conservative and escape `&` too.